CMakeLists.txt

```diff
@@ -228,7 +228,7 @@ tidy_gui_variables()
 
 set(OPENCMISS_MPI ${OPENCMISS_MPI} CACHE STRING "Specify the MPI mnemonic.")
 set(OPENCMISS_BUILD_OWN_MPI NO CACHE BOOL "Build own MPI if system MPI is unavailable.")
-set(OPENCMISS_INSTRUMENTATION "none" CACHE STRING "Specify the instrumentation.")
+set(OC_INSTRUMENTATION "none" CACHE STRING "Specify the instrumentation.")
```

Comment:
The cache variable should be OPENCMISS_INSTRUMENTATION.  The variable used to manipulate this through the command line should be OC_INSTRUMENTATION.

Reply:
?Isn't it the other way around? OPENCMISS_xxx on the command line? (I don't mind if it is OC_xxx on the command line or OPENCMISS_xxx - just that we should be consistent)

Reply:
I think all other flags are OPENCMISS_xxx in the command line.

Reply:
It's not a problem to change the name, just want to be consistent as Chris said. I can use the same variable throughout the files, though